src/thriveopedia/pages/ThriveopediaEvolutionaryTreePage.cs

```diff
@@ -0,0 +1,158 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Godot;
+
+/// <summary>
+///    Thriveopedia page displaying an evolutionary tree of all species in the current game.
+/// </summary>
+/// <remarks>
+///   <para>
+///     Note a lot of this functionality is duplicated from AutoEvoExploringTool.
+///   </para>
+/// </remarks>
+public class ThriveopediaEvolutionaryTreePage : ThriveopediaPage
+{
+    [Export]
+    public NodePath DisabledInFreebuildPath = null!;
+
+    [Export]
+    public NodePath EvolutionaryTreePath = null!;
+
+    [Export]
+    public NodePath SpeciesDetailsLabelPath = null!;
+
+    [Export]
+    public NodePath SpeciesPreviewPath = null!;
+
+    [Export]
+    public NodePath HexPreviewPath = null!;
+
+    private readonly List<Dictionary<uint, Species>> speciesHistoryList = new();
+    private VBoxContainer disabledInFreebuild = null!;
```

Comment:
```suggestion
    private readonly List<Dictionary<uint, Species>> speciesHistoryList = new();

    private VBoxContainer disabledInFreebuild = null!;
```
I prefer blank lines between *completely* different categories of variables for easier visual parsing of what's going on here.